src/components/ChangePlan/PlanCalculator.js

```diff
@@ -445,18 +446,27 @@ const PlanCalculator = ({ location, dependencies: { planService, appSessionRef }
                   >
                     {_('plan_calculator.button_back')}
                   </Link>
-
-                  <a
-                    className="dp-button button-medium primary-green"
-                    href={planService.getBuyUrl(
-                      _('common.control_panel_section_url'),
-                      planData.plan.id,
-                      planData.discount?.id,
-                      safePromoId,
-                    )}
-                  >
-                    {_('plan_calculator.button_purchase')}
-                  </a>
+                  <div className="dp-tooltip-container">
+                    <Link
+                      className="dp-button button-medium primary-green"
+                      disabled={`${currentPlan.idPlan === planData.plan.id ? 'disabled' : ''}`}
+                      to={planService.getBuyUrl(
+                        _('common.control_panel_section_url'),
+                        planData.plan.id,
+                        planData.discount?.id,
+                        safePromoId,
+                      )}
+                    >
+                      {_('plan_calculator.button_purchase')}
+                    </Link>
+                    <S.HiddenTooltip
+                      className={`dp-tooltip-top ${
+                        currentPlan.idPlan !== planData.plan.id ? 'disabled' : ''
+                      }`}
+                    >
+                      <span> {_('plan_calculator.button_purchase_tooltip')}</span>
+                    </S.HiddenTooltip>
+                  </div>
```

Comment:
I'm thinking that maybe, this could be in another component (it will be also reusable for tab tooltips that we already have, in this case, property hidden it will be always `false`)

```
<TooltipContainer  
  hidden={currentPlan.idPlan !== planData.plan.id} 
  content={_('plan_calculator.button_purchase_tooltip')}
>
  <Link
    className="dp-button button-medium primary-green"
    disabled={`${currentPlan.idPlan === planData.plan.id ? 'disabled' : ''}`}
    to={planService.getBuyUrl(
      _('common.control_panel_section_url'),
      planData.plan.id,
      planData.discount?.id,
      safePromoId,
    )}
    >
    {_('plan_calculator.button_purchase')}
  </Link>
</TooltipContainer>
```

![image](https://user-images.githubusercontent.com/10213170/123798828-fbcf0880-d8bd-11eb-98ed-61db0e576e92.png)